content/applications/productivity/studio/pdf_reports.rst

```diff
@@ -232,6 +232,36 @@ fields if needed. Then, specify the default value and press `Enter`.
 .. image:: pdf_reports/powerbox-field.png
    :alt: Select a related field.
 
+Add a data table
+****************
+
+Data tables are used to display :ref:`relational fields <studio/fields/relational-fields>`. To add
+a data table, type `/`, select the :guilabel:`Dynamic Table` command, and select the relation to be
+displayed in the table.
+
+.. note::
+   Only relations of type `one2many` or `many2many` can be displayed as data tables.
+
+Once the table has been added, you can add columns using the table control UI. Position the cursor
+on top of the column, then click the purple rectangle and select an option.
+
+.. image:: pdf_reports/table-add-column.png
+   :alt: Add a column in a dynamic table.
+
+You can then insert the :ref:`field of your choice
+<studio/pdf-reports/add-field>` in the columns. The dialog box that opens shows the source object
+for the field (e.g., the *Tag* model) and the list of available fields.
+
+.. image:: pdf_reports/dynamic-table.png
+   :alt: List of available fields for the Tag model.
+
+.. note::
+   - The default row automatically iterates over the field's content, generating a row on the report
+     for each field value (e.g., one row per tag).
+   - Any row you add using the table control UI is added at the end of the table, but you can move
```

Comment:
not sure i'd mention this at all; the table tools give the option to 'insert above' or 'insert below' anyway - normally the tool should be self explanatory